Sekoia.io/sekoiaio/triggers/base.py

```diff
@@ -163,8 +164,20 @@ def handler_dispatcher(self, raw_message: str):
             return
 
         # We can only manage v2 events
-        if str(message.get("metadata", {}).get("version")) != "2":
-            self.log("Received event with version not handled by the trigger", level="info", event=message)
+        event_version = message.get("metadata", {}).get("version")
+        if event_version is None:
+            # Bypass the version check for backward compatibility
+            # Looks like in some cases version might be missing.
+            self.log(
+                "Received event without version specified. Default version is `2`.", level="warning", event=message
```

Comment:
I prefer that we use `Expected` here as `Default` can be interpreted as `we set the version to 2 for you as you didn't supply one`.
```suggestion
                "Received event without version specified. Expected version is `2`.", level="warning", event=message
```